src/components/ChartCell.js

```diff
@@ -0,0 +1,92 @@
+// @flow
+'use strict';
+
+import React, { PropTypes } from 'react';
+import {
+  StyleSheet,
+  Text,
+  TouchableHighlight,
+  View
+} from 'react-native';
+import Icon from 'react-native-vector-icons/Ionicons';
+import Colors from '../styles/Colors';
+import FontStyles from '../styles/FontStyles';
+
+const displayText = false;
+
+const ChartCell = (props) => {
+  var favIcon = 'ios-heart-outline';
+  if(props.vfrChart.isFavorited == true){
+    favIcon = 'ios-heart';
+  }
+      return (
+        <View style={styles.inputsContainer}>
+            <View style={styles.vfrText}>
+              <TouchableHighlight style={styles.textHighlight} onPress={() => console.log('eventually display more map information')}>
```

Comment:
I think this `TouchableHighlight` encompass all of this view so they can press anywhere in the view and go to the chart view. Also it should have `underlayColor={Colors.primary}`